### PR TITLE
Create new password for users. 

### PR DESCRIPTION
**Pull request by juhangsin**
This PR is for invited user to be able to create a password.

### CODE REVIEW

File: src/user/controller/password.ts [hunk 1]
```diff
@@ -3,9 +3,35 @@ import { Context } from '@curveball/core';
...
+    if (user.type !== 'user') {
+      throw new Forbidden('You can only create passwords for users');
+    }
+
+    await UserService.createPassword(user, userNewPassword);
+
+    ctx.session = {
+      user: user,
+    };
+    log(EventType.changePasswordSuccess, ctx);
```
Comment (juhangsin) [inline src/user/controller/password.ts:29]: Should I create `EventType.createPasswordSuccess` for this?

Comment (evert) [reply src/user/controller/password.ts:29]: I don't really think this endpoint is needed. `PUT` can be used for creating or modifying, and we don't really want the client to have to decide whether it should be created or modified. The only thing really needed to fix this bug is to change the `UPDATE` query to `INSERT ... ON DUPLICATE KEY UPDATE`

**Comment by evert** [comment]
I don't really think this endpoint is needed. PUT can be used for creating or modifying, and we don't really want the client to have to decide whether it should be created or modified. The only thing really needed to fix this bug is to change the UPDATE query to INSERT ... ON DUPLICATE KEY UPDATE.

`POST` is not strictly for creation. We actually prefer `PUT` for creating resources _if_ the client can know the URI of the newly created resource.

**Comment by juhangsin** [comment]
> I don't really think this endpoint is needed. PUT can be used for creating or modifying, and we don't really want the client to have to decide whether it should be created or modified. The only thing really needed to fix this bug is to change the UPDATE query to INSERT ... ON DUPLICATE KEY UPDATE.
> 
> `POST` is not strictly for creation. We actually prefer `PUT` for creating resources _if_ the client can know the URI of the newly created resource.

**Review by BeckyPollard — APPROVED** [review]
This works great Ju! 🙂

**Pull request merged**